pkg/cmd/build/errors_test.go

```diff
@@ -0,0 +1,181 @@
+package build
```

Comment:
I've added some boring tests here, not that they add huge value but it feels the right thing to do.